.travis.yml

```diff
@@ -5,39 +5,54 @@ branches:
     - master
     - /^maintenance_[0-9]+\.[0-9]+\.x$/
 
+sudo: false
+
 matrix:
   include:
     # do one build run with our minimum dependencies
     # (or, well, at least the lowest version number that is available through
     #  anaconda..)
+    #- os: linux
+    #  env: PYTHON_VERSION=2.7 MINIMUM_DEPENDENCIES="True"
+    #- os: linux
+    #  env: PYTHON_VERSION=2.7
+    #- os: linux
+    #  env: PYTHON_VERSION=3.3
+    #- os: linux
+    #  env: PYTHON_VERSION=3.4
+    #- os: linux
+    #  env: PYTHON_VERSION=3.5
     - os: linux
-      env: PYTHON_VERSION=2.7 MINIMUM_DEPENDENCIES="True"
-    - os: linux
-      env: PYTHON_VERSION=2.7
-    - os: linux
-      env: PYTHON_VERSION=3.3
-    - os: linux
-      env: PYTHON_VERSION=3.4
-    - os: linux
-      env: PYTHON_VERSION=3.5
-
-    - os: osx
-      env: PYTHON_VERSION=2.7
-    - os: osx
-      env: PYTHON_VERSION=3.5
-
-sudo: false
+      env: PYTHON_VERSION=3.5 ARCHITECTURE_32BIT="True"
+      sudo: true
+    #- os: osx
+    #  env: PYTHON_VERSION=2.7
+    #- os: osx
+    #  env: PYTHON_VERSION=3.5
 
 install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       export OS="MacOSX";
     else
       export OS="Linux";
     fi
+  # OPT used by setuptools patches contained in newer numpy versions
+  - |
+      if [[ "$ARCHITECTURE_32BIT" == "True" ]]; then
+        export ARCH=""
+        export OPT="-m32"
+        ls /etc/dpkg/dpkg.cfg.d/
+        sudo sh -c "echo 'foreign-architecture i386' > /etc/dpkg/dpkg.cfg.d/multiarch"
```

Comment:
just checked, seems to be on by default. However, need to be cautious, if there are changes in future versions.